src/SALib/analyze/radial_st.py

```diff
@@ -0,0 +1,119 @@
+import numpy as np
+from scipy.stats import norm
+from typing import Dict, Optional
+
+from ..util import ResultDict
+
+__all__ = ['analyze']
+
+def analyze(problem: Dict, Y: np.array, sample_sets: int,
+            num_resamples: int = 1000,
+            conf_level: float = 0.95,
+            seed: Optional[int] = None) -> Dict:
+    """Global Sensitivity Index for `radial` approach.
+
+    Sample size must be of a compatible number, or else results
```

Comment:
What is a compatible number? Can you provide guidance in the docstring here?

Reply:
Modified docstring to be clearer, with references